README.rst

```diff
@@ -0,0 +1,71 @@
+****************
+Caliper Tracking
+****************
+
+
+Description
+###########
+
+Caliper Tracking can be used to transform the edx traditional event logs into `IMS Caliper Standard <http://imsglobal.org/sites/default/files/caliper/v1p1/caliper-spec-v1p1/caliper-spec-v1p1.html>`_ format. Then those logs can be used with any analytics application that is compatible with Caliper Standard events.
+
+Installation
+############
+
+To install **caliper-tracking** in Openedx add the following inside your
+``edx-platform/requirements/edx/base.txt``::
+
+    caliper-tracking==0.9.0
+
+
+Usage
+#####
+
+To enable and use `caliper-tracking`
+
+Add/Enable the `ENABLE_EVENT_CALIPERIZATION` flag in `FEATURES` in the following files:
```

Comment:
Set the value of `ENABLE_EVENT_CALIPERIZATION` flag

Reply:
Done